### ISSUE TITLE
Is this extension working with EWARM version 8.30.1?

### ISSUE THREAD

**Issue by PSLIndigo** [politoleo/iar#14]
I used and loved this extension, but now I need to use EWARM version 8.30.1 and I did not manage to get this extension run successfully with with acutally newest version of EWARM.

**Comment by jbuckAI** [comment]
I haven't been able to get this extension to work for 8.30.2. CTRL+SHIFT+B only opens the command options and VSCode is recommending to create a tasks.json file

**Comment by jbuckAI** [comment]
I did put my "path" in iar.json as 
    "path": "C:\\Program Files (x86)\\IAR Systems\\Embedded Workbench 8.1\\",

**Comment by politoleo** [comment]
Yes It is, be sure to enable it using 
`iar.enable = true`
in settings.

**Comment by PSLIndigo** [comment]
Sorry for the late reply. I had to focus on a development sprint first. 

I figured out, that the rootcause must be in the settings of my project's "Debug" build configuration. Using this configuration in the iar.json file, i get "make failed". This build configuration is working fine, when compiling from EWARM.

Using the "Release" configuration in iar.json, compiling and creating c_cpp_properties.json entries works fine.
